src/assignments/flask-game.rst

```diff
@@ -10,14 +10,16 @@ from start to finish.
 Now that you have some Flask experience, your team can move the game out of the
 console and into a browser!
 
-.. admonition:: Example
+.. figure:: figures/RPSLS.gif
+   :alt: On left console version of rock paper scissors, on the right the flask version.
+   :height: 500px
```

Comment:
Cool!  I like the new figure.  However, let's move it inside the Example admonition (below the text) so that the image/GIF has some context.

![Screen Shot 2021-08-03 at 6 19 31 PM](https://user-images.githubusercontent.com/31870126/128098384-2cfd4962-d0bc-4ae4-a4aa-b6a8bdc44aa0.png)

One last tip: To accommodate different sized screens, I've tried to avoid setting the `:height:` in the `.rst` files to a specific number of pixels.  Instead, I've set the width to a percentage of the window (or admonition) size.  For example, `:width: 80%`.  Let's use this format for your image as well.

After those tweaks, merge away!

Reply:
Done!